tests/models/opt/test_modeling_opt.py

```diff
@@ -348,8 +348,8 @@ def test_generation(self):
         GEN_OUTPUT = []
 
         tokenizer = GPT2Tokenizer.from_pretrained("patrickvonplaten/opt_gpt2_tokenizer")
-        for model in self.all_model_path:
```

Comment:
Actually can you revert this one? I'll correct this test here: https://github.com/huggingface/transformers/pull/17228